File: instagram_graph_api/business_discovery.php
<?php
	include 'defines.php';

	// get instagram user metadata endpoint
	$endpointFormat = ENDPOINT_BASE . '{ig-user-id}?fields=business_discovery.username({ig-username}){username,website,name,ig_id,id,profile_picture_url,biography,follows_count,followers_count,media_count}&access_token={access-token}';
	$endpoint = ENDPOINT_BASE . $instagramAccountId;

	// username
	$username = 'IG-USER-NAME';

	// endpoint params
	$igParams = array(
		'fields' => 'business_discovery.username(' . $username . '){username,website,name,ig_id,id,profile_picture_url,biography,follows_count,followers_count,media_count}',
		'access_token' => $accessToken
	);

	// add params to endpoint
	$endpoint .= '?' . http_build_query( $igParams );

	// setup curl
	$ch = curl_init();
	curl_setopt( $ch, CURLOPT_URL, $endpoint );
	curl_setopt( $ch, CURLOPT_SSL_VERIFYHOST, false );
	curl_setopt( $ch, CURLOPT_SSL_VERIFYPEER, false );
	curl_setopt( $ch, CURLOPT_RETURNTRANSFER, true );

	// make call and get response
	$response = curl_exec( $ch );
	curl_close( $ch );
	$responseArray = json_decode( $response, true );
?>
<!DOCTYPE html>
<html>
	<head>
		<title>Getting an Instagram Users Metadata</title>
		<meta charset="utf-8" />
	</head>
	<body>
		<h1>Getting an Instagram Users Metadata</h1>
		<hr />
		<br />
		<div style="display:inline-block;width:100%">
			<div style="float:left">
				<img style="width:50px;border-radius:50%;" src="<?php echo $responseArray['business_discovery']['profile_picture_url']; ?>" />
			</div>
			<div style="float:left;margin-left:20px">
				<a target="_blank" href="https://www.instagram.com/<?php echo $responseArray['business_discovery']['username']; ?>">
					<h3><?php echo $responseArray['business_discovery']['username']; ?></h3>
				</a>
				<div style="display:inline-block">
					<b><?php echo $responseArray['business_discovery']['media_count']; ?></b> posts
				</div>
				<div style="display:inline-block;margin-left:20px">
					<b><?php echo $responseArray['business_discovery']['followers_count']; ?></b> followers
				</div>
				<div style="display:inline-block;margin-left:20px">
					<b><?php echo $responseArray['business_discovery']['follows_count']; ?></b> following
				</div>
			</div>
		</div>
		<div>
			<h4><?php echo $responseArray['business_discovery']['name']; ?></h4>
			<div>
				<?php echo nl2br( $responseArray['business_discovery']['biography'] ); ?>
			</div>
			<div>
				<a target="_blank" href="<?php echo $responseArray['business_discovery']['website']; ?>">
					<h3><?php echo $responseArray['business_discovery']['website']; ?></h3>
				</a>
			</div>
		</div>
		<br />
		<hr />
		<h3>Endpoint: <?php echo $endpointFormat; ?></h3>
		<hr />
		<h3>Raw Response</h3>
		<textarea style="width:100%;height:300px;"><?php print_r( $responseArray ); ?></textarea>
	</body>
</html>

File: instagram_graph_api/python/business_discovery.py
from defines import getCreds, makeApiCall

def getAccountInfo( params ) :
	""" Get info on a users account
	
	API Endpoint:
		https://graph.facebook.com/{graph-api-version}/{ig-user-id}?fields=business_discovery.username({ig-username}){username,website,name,ig_id,id,profile_picture_url,biography,follows_count,followers_count,media_count}&access_token={access-token}

	Returns:
		object: data from the endpoint

	"""

	endpointParams = dict() # parameter to send to the endpoint
	endpointParams['fields'] = 'business_discovery.username(' + params['ig_username'] + '){username,website,name,ig_id,id,profile_picture_url,biography,follows_count,followers_count,media_count}' # string of fields to get back with the request for the account
	endpointParams['access_token'] = params['access_token'] # access token

	url = params['endpoint_base'] + params['instagram_account_id'] # endpoint url

	return makeApiCall( url, endpointParams, params['debug'] ) # make the api call

params = getCreds() # get creds
params['debug'] = 'no' # set debug
response = getAccountInfo( params ) # hit the api for some data!

print "\n---- ACCOUNT INFO -----\n" # display latest post info
print "username:" # label
print response['json_data']['business_discovery']['username'] # display username
print "\nwebsite:" # label
print response['json_data']['business_discovery']['website'] # display users website
print "\nnumber of posts:" # label
print response['json_data']['business_discovery']['media_count'] # display number of posts user has made
print "\nfollowers:" # label
print response['json_data']['business_discovery']['followers_count'] # display number of followers the user has
print "\nfollowing:" # label
print response['json_data']['business_discovery']['follows_count'] # display number of people the user follows
print "\nprofile picture url:" # label
print response['json_data']['business_discovery']['profile_picture_url'] # display profile picutre url
print "\nbiography:" # label
print response['json_data']['business_discovery']['biography'] # display users about section